marklogic-data-hub/src/main/java/com/marklogic/hub/mapping/TestMapping.java

```diff
@@ -0,0 +1,28 @@
+package com.marklogic.hub.mapping;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.marklogic.client.DatabaseClient;
+import com.marklogic.client.extensions.ResourceManager;
+import com.marklogic.client.io.JacksonHandle;
+import com.marklogic.client.util.RequestParameters;
+
+public class TestMapping extends ResourceManager {
```

Comment:
Let's use data services here instead (I was just looking at MappingValidator and thinking how easy that would be to switch over). 

There's now a data service on the develop branch. See around line 160 for the Gradle task to generate a Java interface based on an API file. 

I recommend generating a "com.marklogic.hub.dataservices.MappingService" and calling this function "testMapping". We can add validateMapping at a later date. 

Let me know if you have any questions, I can help out and/or submit a PR into this to do the work.

Reply:
I'm actually going to submit a PR for the data services change, as I'd like to fold MappingValidator into it as well. Should be ready shortly.